packages/gdl-frontend/components/Navbar/index.js

```diff
@@ -51,7 +51,6 @@ const Navbar = ({ onMenuClick, menuIsExpanded }: Props) => {
         <IconButton
           color="inherit"
           onClick={onMenuClick}
-          aria-expanded={menuIsExpanded}
```

Comment:
This should also be removed from the prop types and not passed to the component at all any more, since it isn't in use